### PR TITLE
COMMERCE-5505 | COMMERCE-5506 | COMMERCE-5508 | COMMERCE-3764

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:forward

File: modules/apps/commerce/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -73,6 +73,142 @@ in ascending chronological order.
 
 ## Breaking Changes List
 
+### Product Card and Product Rendering Strategies in storefront
```
Comment (dsitu) [inline modules/apps/commerce/BREAKING_CHANGES.markdown:76]:
```suggestion
### Product Card and Product Rendering Strategies in Storefront
```

**Review by dsitu — CHANGES_REQUESTED** [review]
I think we should sort the newest changes towards the bottom

**Comment by ivicac** [comment]
ci:stop

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@gianmarcobrunialti can you, please, rebase this PR? thx

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**